### PR TITLE
Make it explicit we are currently using a signed char

### PR DESCRIPTION
**Pull request by ktf**
On x86 Linux `char` is assumed to be signed, however that's an implementation detail.

This makes sure we are explicit about the sign-ness.

### CODE REVIEW

**Comment by ktf** [comment]
@davidrohr this fixes one of the two tests on ARM.

**Review by davidrohr — DISMISSED** [review]
makes sense, but the error during the RTC test is genuine :(.

**Comment by ktf** [comment]
Added the specialization for the printout.

File: GPU/GPUTracking/utils/qconfig_helpers.h [hunk 1]
```diff
@@ -42,6 +42,11 @@ inline std::string print_type<char>(char val)
   return std::to_string(val);
 };
 template <>
+inline std::string print_type<signed char>(char val)
```
Comment (davidrohr) [inline GPU/GPUTracking/utils/qconfig_helpers.h]: should be `signed char val` to be consistent :)

Comment (ktf) [reply GPU/GPUTracking/utils/qconfig_helpers.h]:
```suggestion
inline std::string print_type<signed char>(signed char val)
```

**Comment by ktf** [comment]
Verified to fix one of the two ARM tests.

**Pull request merged**